products/mozilla_org/extract_strings.md

```diff
@@ -0,0 +1,79 @@
+
+## Step by step details for the entire process
+
+### Extract strings from a template in Bedrock
+
+The process is detailed in [this document](../../products/mozilla_org/working_bedrock.md).
+
+Note: you only use this generated .lang file as a reference. You still need to add comments and reorder strings before exposing them for localization.
+
+### Prepare the en-US .lang file
+
+It is very important to have a clean, error free .lang file in en-US. Once the file is approved and content is pushed into other locales, if an error is found then, it will take much more effort to correct. If localizers have already started their work, the cleanup is even messier. At that point, it is a fix in one file versus fixes in 100+ files.
+
+Initial review should include checking the file against the page on the testing server and checking against the template. This is to ensure that there are no hard coded strings.
+
+Note: if there is an update to an existing page, it is advised to still generate the .lang file, and copy the new string there instead of copying it from the .html file. This is to make sure, for example, that the syntax used for variable replacements is correct, and remove eventual extra white spaces between words that are not carried over to the .lang file.
+
+### Add notes and comments to the extracted file:
+
+Add important information at the top of the file:
+  `## NOTE: this is a note.`
+* A Note for [demo/test URLs](https://github.com/mozilla-l10n/www.mozilla.org/blob/master/en-US/firefox/facebookcontainer/index.lang).
+* A Note for test instruction if necessary, for a page that is more dynamic, such as the [privacy_protection_tour.lang](https://github.com/mozilla-l10n/www.mozilla.org/blob/master/en-US/firefox/tracking-protection-tour.lang).
+* Make sure to add tags, if there are strings [bound to l10n tags](working_bedrock.md#l10n-tags).
+
+Proceed to the content of the file. Add a comment to the following type of strings to provide more context:
+* Page title and description. Typically, a page title string should be translated in a short or catchy way
+* If the string contains a link, to which [URL](https://github.com/mozilla-l10n/vm_scripts/blob/master/update_mozillaorg.sh0 it points to. This gives context to localizers.
+* If a string is behind a tag, make sure to add the [tag binding](https://github.com/mozilla-l10n/langchecker/wiki/.lang-files-format#string-meta-data).
+* A new string behind a tag makes another string obsolete. Make sure to add [an obsolete comment](https://github.com/mozilla-l10n/vm_scripts/blob/master/update_mozillaorg.sh) to help localizers prioritize their work, and that old and new strings are next to each other. This will also make removal of old strings easier in further updates.
```

Comment:
Did you want to put a link here? Because the URL is definitely wrong.

Reply:
Quite a few of them were wrong. I hope I caught them all.